### ISSUE TITLE
Add new unknown states for Audio/Motion/Flashing Hazards

### ISSUE THREAD

**Issue by clapierre** [w3c/a11y-discov-vocab#92]
There is a need to specify if a particular Hazard hasn't been considered if that type of content exists in a Publication.

For example if a publication has no Video content the author can clearly state
`noMotionSimulationHazard`
and
`noFlashingHazard`

However if that same publication includes an audio clip which hasn't been "examined" to know if it has or doesn't have any potential sound hazards, the only option would be not to include that statement in the accessibility Metadata, instead of clearly pointing out that the author doesn't know if a hazard exists or not.

Recommendation as discussed in the 7/17/2023 Editors call is to add 3 new unknown values to the Hazard metadata

`unknownFlashingHazard`, `unknownMotionSimulationHazard`, and `unknownSoundHazard`

The following two documents should be updated
[https://w3c.github.io/a11y-discov-vocab/index.html](https://w3c.github.io/a11y-discov-vocab/index.html)
and
[https://w3c.github.io/a11y-discov-vocab/crosswalk/index.html](https://w3c.github.io/a11y-discov-vocab/crosswalk/)

**Comment by mattgarrish** [comment]
Closing with #93
